README.md

```diff
@@ -202,34 +211,34 @@ iscsc.fr
 │   ├── Dockerfile          *Dockerfile to build the frontend container*
 │   └── src                 *source code of the website*
 │       ├── components/         *source code of main components of the website*
-│       ├── context/            *defines the context function to keep track data with useReducer*
+│       ├── context/            *defines the context function to keep track of data with useReducer*
 │       ├── hooks/              *defines the hooks that trigger the context functions*
 │       ├── pages/              *source code of the pages of the website*
 │       ├── App.js              *defines the routes of the application*
 │       ├── index.js            *main js application of the website*
 │       └── index.css           *css styling file of the website*
 ├── scripts
-│   └── gpg-share.sh        *share a secret files to others with pgp keys*
+│   └── gpg-share.sh        *share a secret file to others with pgp keys*
 │
 ├── nginx               *reverse proxy server for the production mode*
 │   ├── Dockerfile          *Dockerfile to build the nginx container*
 │   ├── run_nginx.sh        *script to generate the nginx conf from the template*
 │   └── nginx.conf.template *template for the nginx conf, needs to be filled with env variables*
 │
-├── bump.sh             *script used to bump version of frontend, backend and whole website*
+├── bump.sh             *script used to bump version of frontend, backend, and whole website*
 ├── docker-compose.yml  *docker compose config file to deploy the website in production mode*
-├── package.json        *contains the current version and informations about the website*
+├── package.json        *contains the current version and information about the website*
 └── README.md           *this file*
 ```
 
-## Bugs and recommendations
+## 4. Bugs and recommendations
 
-Because this website is still in development, do not hesitate to open an issue if you experience any troubles when using it.
+Because this website is still in development, do not hesitate to open an issue if you experience any trouble using it.
 Also, feel free to share your recommendations regarding the color scheme, routes, design, UX, etc...
 
-## Incoming works
+## 5. Incoming works
```

Comment:
Shouldn't it be
```suggestion
## 5. Upcoming work
```
?
At least I'm pretty sure about the **work**
the TOC would have to be be updated too if you agree!

Reply:
if it's "future work" then it's "upcoming work" :+1:

Reply:
@OlaPom you did not correct the ~`works`~ --> `work` but maybe it is on purpose and I'm mistaken?

Reply:
Also I must warn you, this is considered a bad reviewing habit to resolve the thread yourself :confused: generally we prefer to let the one that opened it close it :wink:

Reply:
Sorry for that! I wanted to clean up, but I need to learn more about GitHub etiquette. :blush: I corrected the heading.

Reply:
No problem! and tks for the modification 😉